src/owlwatch/README.md

```diff
@@ -1,30 +1,24 @@
 # owlwatch
 
-Tool for comparing panels and mappings. A data model is defined in [schema module](schema/model.py). This model is based
-on a **Schema** abstraction, used as a common definition to compare ES Mappings and Kibana panels as they are exported by
-[Kidash](https://github.com/grimoirelab/GrimoireELK/tree/master/kidash) tool.
+Tool for comparing panels and mappings. A data model is defined in [schema module](schema/model.py). This model is basedon a **Schema** abstraction, used as a common definition to compare ES Mappings and Kibana panels as they are exported by [Kidash](https://github.com/grimoirelab/GrimoireELK/tree/master/kidash) tool.
 
 ## Supported Versions
 Tested for:
  * ElasticSearch: 5.4.0
  * Lucene: 6.5.0
  * Kibana 5: 5.1.1-SNAPSHOT
 
```

Comment:
Pretty strange combination. ElasticSearch: 5.4.0 won't work with Kibana 5: 5.1.1-SNAPSHOT (Kibiter 5.1.1). Are you sure about this requirements?